trl/scripts/sft.py

```diff
@@ -109,7 +110,14 @@ def main(script_args, training_args, model_args):
     ################
     # Dataset
     ################
-    dataset = load_dataset(script_args.dataset_name, name=script_args.dataset_config)
+    print("loading: {0}".format(script_args.dataset_name))
+    
+    if os.path.exists(script_args.dataset_name) == True:
+        print("loading dataset from local file")
+        dataset = load_dataset("json", data_files=script_args.dataset_name)
+    else:
+        print("loading dataset from online")
+        dataset = load_dataset(script_args.dataset_name, name=script_args.dataset_config)
```

Comment:
```suggestion
    if os.path.exists(script_args.dataset_name):
        dataset = load_dataset("json", data_files=script_args.dataset_name)
    else:
        dataset = load_dataset(script_args.dataset_name, name=script_args.dataset_config)
```

Reply:
sure. good idea. sounds to be a much more better coding practice!

Reply:
sounds to be fixed. I put more content alas.